Src/FluentAssertions/Specialized/DelegateAssertionsBase.cs

```diff
@@ -49,6 +50,30 @@ protected ExceptionAssertions<TException> ThrowInternal<TException>(Exception ex
         return new ExceptionAssertions<TException>(expectedExceptions);
     }
 
+    private protected ExceptionAssertions<TException> ThrowInternal<TException>(
```

Comment:
🔧 This method belongs in the `AsyncFunctionAssertions`,

Reply:
This does not work because it is accessing `private readonly IExtractExceptions extractor`.

Reply:
The `AsyncFunctionAssertions` class also has that `IExtractExceptions` as a parameter.